src/Components/v2/AuctionFAQ.tsx

```diff
@@ -0,0 +1,184 @@
+            All bidders need to register by completing an online form and
+            providing all required information, such as their full name, contact
+            information, and credit card details. Auction Houses often require
+            additional vetting of bidders and may prevent registration in
+            advance of a sale. Check the instructions for the sale you are
+            interested in and register early. We will communicate the status of
+            your registration via the email and contact information you provide.
+          </Serif>
+          <Spacer mb={2} />
+          <Serif size="3" weight="semibold">
+            How do I place a bid?
+          </Serif>
+          <Serif size="3">
+            Bidding on Artsy is easy with our automatic bidding system. Select
+            your bid amount as long as it is greater than or equal to the next
+            minimum bid shown on the bidding screen, then click the “Bid”
+            button. This will automatically place a bid for you at the next
+            increment and save the amount you entered (if higher) as your
+            “Maximum Bid.”
+          </Serif>
+          <Spacer mb={2} />
+          <Serif size="3" weight="semibold">
+            What is a Maximum Bid?
+          </Serif>
+          <Serif size="3">
+            If you enter a bid amount higher than the next minimum bid, the
+            amount you enter is treated as your “Maximum Bid.” Entering a
+            Maximum Bid does not necessarily mean you will pay that price, and
+            you may pay less. As the auction progresses, our system will bid
+            automatically for you against other bidders, according to our
+            automatic bidding increments (see below), up to (but not exceeding)
+            your Maximum Bid, only as much as necessary to maintain your
+            position as highest bidder. If two bidders attempt to enter the same
+            Maximum Bid, the first bidder to enter that amount will be the
+            winner.
+          </Serif>
+          <Spacer mb={2} />
+          <Serif size="3" weight="semibold">
+            What are Bidding Increments?
+          </Serif>
+          <Serif size="3">
+            Our automatic bidding increments determine the next minimum bid that
+            can be placed. They are based on the current bid of each item and
+            increase at the following intervals, when the current bid is:
+            <br />
+            <br />
+            Under $1,000: $50
+            <br />
+            $1,000 - $1,999: $100
+            <br />
+            $2,000 - $4,999: $250
+            <br />
+            $5,000 - $9,999: $500
+            <br />
+            $10,000 - $19,999: $1,000
+            <br />
+            $20,000 - $49,999: $2,000
+            <br />
+            $50,000 - $99,999: $5,000
+            <br />
+            <br />
+            At or above $100,000: $10,000 Note: Our usual bidding increments are
```

Comment:
should there be some line breakage before the note here?

Reply:
![image](https://user-images.githubusercontent.com/1242537/64877753-ebe32780-d649-11e9-90da-9ad2174abc94.png)
might want to keep it to just one br between the $50,000 line and the 'at or above' line too ?

Reply:
Oh thanks! I misplaced the last break. I'm going to keep 2 breaks for now which is consistent with the previous design, can update later if necessary.